extensions/wifi/internal.m

```diff
@@ -110,14 +58,14 @@ - (void)invokeCallback:(id)object {
         lua_State *L = [skin L] ;
         [[LuaSkin shared] pushLuaRef:refTable ref:_fnRef];
         if ([object isKindOfClass:[NSError class]]) {
-            log_to_console(L, _cINFO, [(NSError *)object localizedDescription]) ;
+            [LuaSkin logInfo:[(NSError *)object localizedDescription]] ;
```

Comment:
This method looks like it has a lot of built-up LuaSkin weirdness. We have a `skin` object at the top, so the second `LuaSkin shared` is unnecessary, and these log calls could use the instance variants too.